### PR TITLE
Cleanup TS files, simplify and export every functions to window

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | A small cleanup was required, simplified some things and exported every useful functions for modules/child theme uses
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | You could probably verify the window object but no tests are required

### CODE REVIEW

**Review by matks — COMMENTED** [review]
Looks OK to me 👍

**Comment by NeOMakinG** [comment]
@Hlavtox @mparvazi when you've some time, feel free to review/stamp this one, I guess it's pretty easy and fast to merge :)

**Comment by mparvazi** [comment]
@NeOMakinG 

There is three type castings here: `src/js/components/useQuantityInput.ts`

```javascript
const qtyInputNodeList = document.querySelectorAll(selector) as NodeListOf<HTMLElement>;
const incrementButtonIcons = incrementButton.querySelectorAll('i') as NodeListOf<HTMLElement>;
const decrementButtonIcons = decrementButton.querySelectorAll('i') as NodeListOf<HTMLElement>;

```

File: src/js/constants/selectors-map.ts [hunk 1]
```diff
@@ -111,10 +111,6 @@ export const visiblePassword = {
   visiblePassword: '.js-visible-password',
 };
 
-export const dropdownToggles = {
```
[the remark this answers is not in the document]
Comment (NeOMakinG) [reply src/js/constants/selectors-map.ts:114]: Alright, depending on the one getting merged first, the other will have to rebase!

File: src/js/constants/selectors-map.ts [hunk 2]
```diff
@@ -131,6 +131,7 @@ const selectorsMap = {
   modalBody: '.modal-body',
   pageCms: '.page-cms',
   quickview: '.js-quickview',
+  menuDropdownToggles: '.js-menu-desktop .dropdown .dropdown-toggle[data-depth]',
```
Comment (mparvazi) [inline src/js/constants/selectors-map.ts:134]: needed?

Comment (NeOMakinG) [reply src/js/constants/selectors-map.ts]: removed, was a mergefix mistake!

**Comment by NeOMakinG** [comment]
@mparvazi a bit tired of rebasing every weeks, let's merge this as all your feedbacks has been added

**Pull request merged**